elements/howto-menu/demo.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+
+<style>
+  /* The tooltip is by default unstyled. */
+  howto-menu {
+    position: absolute;
+    display: block;
+    width: 10em;
+  }
+
+  [role="menuitem"] {
+    display: block;
+    width: 100%;
+    padding: .2em .5em;
+    border: 1px solid lightgrey;
+  }
+
+  /* The menu is closed. */
+  howto-menu[aria-hidden="true"] {
+    display: none;
+  }
+</style>
+
+<div class="text">
+   <button id="menu-btn">open menu</button>
+   <howto-menu aria-labelledby="menu-btn">
+     <span role="menuitem">first option</span>
+     <span role="menuitem">second option</span>
+     <span role="menuitem">other option</span>
```

Comment:
I plan to add menuitem, menuitemradio and menuitemcheckbox. Also, there should be a demo where all of them work together. Is it ok to do it in a separate PR though?